test/component_failures_test.py

```diff
@@ -163,7 +163,7 @@ def check_components_alive(self, component_type, check_component_alive):
       if check_component_alive:
         self.assertTrue(component.poll() is None)
       else:
-        self.assertTrue(component.poll() <= 0)
+        self.assertTrue(not component.poll() is None)
```

Comment:
The plasma store now returns 1 if an error occurs (before that the status was negative). This is because of the arrow status checking macros.